javaslang-benchmark/src/test/java/javaslang/benchmark/JmhRunner.java

```diff
@@ -29,20 +29,20 @@ public static void devRunWithAssertions(Class benchmarkClass) {
         runAndReport(benchmarkClass, QUICK_WARMUP_ITERATIONS, QUICK_MEASUREMENT_ITERATIONS, Assertions.Enable);
     }
 
-    private static void runAndReport(Class benchmarkClass, int warmUpIterations, int measurementIterations, Assertions assertions) {
-        Collection<RunResult> results = run(benchmarkClass, warmUpIterations, measurementIterations, assertions);
+    private static void runAndReport(Class benchmarkClass, int warmupIterations, int measurementIterations, Assertions assertions) {
```

Comment:
Quick question: do you deliberately ignore the type of the `Class`, i.e. why no `Class<?>`?